### ISSUE TITLE
IPv6 no longer working

### ISSUE THREAD

**Issue by mikecurry74** [ab77/netflix-proxy#79]
The recent workaround for IPv6 no longer seems to work. Anyone know if they're blocking IP's or ranges?

**Comment by ab77** [comment]
Works fine on two providers I've tried just now. Please do some troubleshooting first to see if the problem is specific to a provider or something to do on your end.

-- ab1

**Comment by mikecurry74** [comment]
thanks for the info, I'll keep at it. Is there a private IRC or something setup anywhere for this type of discussion?

**Comment by ab77** [comment]
Not to my knowledge, best to just keep the conversation here, redacting the
obvious detail.

-- ab1

**Comment by mikecurry74** [comment]
Ok, I've tried a few different places. Following the directions has always worked for me, but I was careful to try again, step-by-step. I am seeing a new message from several hosting providers. I am getting "Whoops, something went wrong..." Internet connection problem. 

https://help.netflix.com/article/en/node/190#steps-for-mac

**Comment by leszczynski-it** [comment]
I had similar issue with domain nflxvideo.net. Adding this domain to zones.override fix the issue for me.

**Comment by ab77** [comment]
FYI: adding `nflxvideo.net` will proxy all video traffic and drive up bandwidth costs.

-- ab1

**Comment by ab77** [comment]
@mikecurry74: from your machine, please run the following commands and let me know what the output is..

```
dig netflix.com
curl -v4Lo /dev/null http://netflix.com/
```

**Comment by leszczynski-it** [comment]
@ab77 Without this I'm receiving 420 error on all video files. This is separate issue?

**Comment by ab77** [comment]
@leszczynski-it: what is the full FQDN of the `nflxvideo.net` (e.g. `ipv6_1-cxl0-c029.1.ewr002.ix.nflxvideo.net`) and what do you get when you request a segment directly with `cURL`?

**Comment by mikecurry74** [comment]
Dig results (truncated)

;; ANSWER SECTION:
netflix.com.		59	IN	A	23.23.191.68
netflix.com.		59	IN	A	54.204.43.31
netflix.com.		59	IN	A	54.204.2.219
netflix.com.		59	IN	A	174.129.2.58
netflix.com.		59	IN	A	75.101.139.66
netflix.com.		59	IN	A	107.20.151.133
netflix.com.		59	IN	A	107.20.177.34
netflix.com.		59	IN	A	50.19.210.42
netflix.com.		59	IN	A	54.225.192.83
netflix.com.		59	IN	A	107.20.154.246
netflix.com.		59	IN	A	54.243.253.96
netflix.com.		59	IN	A	23.21.190.124

;; Query time: 45 msec
;; SERVER: 2001:**********

**Comment by ab77** [comment]
@mikecurry74: you PC isn't using your SmartDNS server if you are getting tat response from dig. You should only get back one answer, which will contain the IP address of your VPS.

**Comment by mikecurry74** [comment]
Ok, I forgot to hit apply. I am getting the IP Now, will update soon.

**Comment by leszczynski-it** [comment]
@ab77 I don't have curl on my windows machine. 
ipv6_1-lagg0-c105.1.nyc001.ix.nflxvideo.net
ipv4_1-lagg0-c107.1.nyc001.ix.nflxvideo.net
ipv4_1-lagg0-c167.1.nyc001.ix.nflxvideo.net

Response headers from chrome (asking for file):
HTTP/1.1 420 OK
Server: nginx
Date: Thu, 03 Mar 2016 20:03:01 GMT
Content-Length: 0
Connection: keep-alive
X-Netflix-Geo-Check: failed
Cache-Control: no-store
Pragma: no-cache
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: X-TCP-Info,X-Session-Info
X-TCP-Info: h0=3502684736;h1=1534387489;h2=2126940727;h3=4179238346;h4=2775455563;

Direct acces to ipv4_1-lagg0-c167.1.nyc001.ix.nflxvideo.net gave me 403.

I can send You headers from cURL later if You want.

**Comment by ab77** [comment]
@leszczynski-it: I guess the answer is in the header `X-Netflix-Geo-Check: failed`. Looks like they are starting to do a geo check on the video delivery endpoint as well, which means the only way around it is to proxy that domain as well or find a VPS that works.

**Comment by mikecurry74** [comment]
I setup a DNS server, forwarded one IP to each domain, all but the CDN's. If a server was blocked, generally I would replace www.netflix.com server, and would fix it for me. Replacing that server with a known good one, no longer works. This is the first time I've seen that header as well. Everything flows through the HAProxy servers without issue, its not until the stream is activated that it hangs with a proxy error. I will be investigating this tonight as well - I think we should move this into a private IRC. Hit me up with an email mikecurry74 at gmail . com

**Comment by ab77** [comment]
#netflix-proxy on freenode..
